app/subvisual-blog/services/post_processor.rb

```diff
@@ -11,13 +11,25 @@ def initialize(post)
     def process
       extensions = { autolink: true, fenced_code_blocks: true, tables: true }
       redcarpet = Redcarpet::Markdown.new(SyntaxHighlightedRenderer, extensions)
-      post.processed_body = redcarpet.render(post.body)
+      post.processed_body = redcarpet.render(body_without_main_headings)
 
       return unless post.respond_to?(:processed_intro=)
 
       post.processed_intro = redcarpet.render(markdown_intro)
     end
 
+    def body_without_main_headings
+      if has_main_headings?
+        post.body.gsub(/^#/, "##")
```

Comment:
Regarding this specific line, is it safe to assume all posts have their main heading prefixed with `#` instead of underlined with `=`?